src/components/VAudioTrack/VWaveform.vue

```diff
@@ -1,23 +1,19 @@
 <template>
   <div
+    v-bind="seekAttributes"
```

Comment:
The seekable attributes should only be binded if the waveform is interactive and has a role of slider. Currently, when it's not a slider (on the audio search page), Lighthouse shows aria attributes mismatch.